contracts/entry-point/src/execute.rs

```diff
@@ -524,6 +524,10 @@ pub fn execute_post_swap_action(
         transfer_out_asset
     };
 
+    response = response
+        .add_attribute("amount_out", transfer_out_asset.amount().to_string())
```

Comment:
let's make this name a little more unique for easier searching, maybe "post_swap_action_amount_out" and "post_swap_action_denom_out"